clients/pkg/promtail/targets/syslog/syslogtarget.go

```diff
@@ -57,19 +61,31 @@ func NewSyslogTarget(
 	handler api.EntryHandler,
 	relabel []*relabel.Config,
 	config *scrapeconfig.SyslogTargetConfig,
+	encodingFormat string,
```

Comment:
nit: I would be more specific that this is text encoding.

```suggestion
	textEncoding string,
```